### PR TITLE
open-cluster-management/backlog#16812 upd role fmt

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: access_control/rbac.adoc [hunk 1]
```diff
@@ -38,16 +38,16 @@ Some product resources are cluster-wide and some are namespace-scoped. You must
...
 
-| *open-cluster-management:admin:managed-cluster-<managed_cluster_name>*
-| A user with cluster binding to the `managed-cluster-<managed_cluster_name>` role has administrator access to the `ManagedCluster` resource named,  `managedcluster-<managed_cluster_name>`. When a user has a managed cluster, this role is automatically created.
-| *open-cluster-management:view:managed-cluster-<managed_cluster_name>*
-| A user with cluster binding to the `managed-cluster-<managed_cluster_name>` role has view access to the `ManagedCluster` resource named,  `managedcluster-<managed_cluster_name>`.
+| *open-cluster-management:admin:<managed_cluster_name>*
+| A user with cluster binding to the `<managed_cluster_name>` role has administrator access to the `ManagedCluster` resource named,  `<managed_cluster_name>`. When a user has a managed cluster, this role is automatically created.
+| *open-cluster-management:view:<managed_cluster_name>*
+| A user with cluster binding to the <managed_cluster_name>` role has view access to the `ManagedCluster` resource named,  `<managed_cluster_name>`.
```
Comment (ldpliu) [inline access_control/rbac.adoc]: missed a ` before <managed_cluster_name>

File: access_control/rbac.adoc [hunk 2]
```diff
@@ -38,16 +38,16 @@ Some product resources are cluster-wide and some are namespace-scoped. You must
...
+| *open-cluster-management:managedclusterset:admin:<managed_clusterset_name>*
+| A user with cluster binding to the `<managed_clusterset_name>` role has administrator access to `ManagedCluster` resource named `<managed_clusterset_name>`. The user also has administrator access to `managedcluster.cluster.open-cluster-management.io`, `clusterclaim.hive.openshift.io`, `clusterdeployment.hive.openshift.io`, and `clusterpool.hive.openshift.io` resources, which has the managed cluster set labels: `cluster.open-cluster-management.io` and `clusterset=<managed_clusterset_name>`. A role binding is automatically generated when you are using a cluster set. See link:../clusters/managedclustersets.adoc#managedclustersets[Creating and managing ManagedClusterSets] to learn how to manage the resource.
```
Comment (ldpliu) [inline access_control/rbac.adoc]: _A user with cluster binding to the `<managed_clusterset_name>` role_ ..... changed to _A user with cluster binding to the `open-cluster-management:managedclusterset:admin:<managed_clusterset_name>` role_ will be better.
and all the description in this table has this problem. the role name in description should be same as the Role in first column

File: access_control/rbac.adoc (hunk 1 above)
Comment (cadawson) [reply access_control/rbac.adoc]: Good catch. Thank you!

File: access_control/rbac.adoc (hunk 2 above)
Comment (cadawson) [reply access_control/rbac.adoc]: Changed these.

**Review by cadawson — COMMENTED** [review]
Made the changes. Thanks @ldpliu !

**Comment by cadawson** [comment]
This should look better, I hope. Thanks!

**Review by ldpliu — APPROVED** [review]

**Comment by ldpliu** [comment]
it's ok for me now, please feel free to merge it.

**Pull request merged**